### PR TITLE
LPS-165571 Update logic to set objectFieldSettings values

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx [hunk 1]
```diff
@@ -314,12 +314,20 @@ export default function ObjectFieldFormBase({
...
+						let newObjectFieldSettings: ObjectFieldSetting[] = [];
+
+						if (values.objectFieldSettings) {
+							newObjectFieldSettings = [
 								...(values.objectFieldSettings?.filter(
 									(objectFieldSetting) =>
 										objectFieldSetting.name !== 'output'
 								) as ObjectFieldSetting[]),
+							];
+						}
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx:326]: Hey @seltong, in this case as you are doing a simple filter and only the filter result will be the `newObjectFieldSettings`, we don't need to use the [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax), just pass the filter return directly:

```tsx
newObjectFieldSettings = values.objectFieldSettings?.filter(
    (objectFieldSetting) =>
	    objectFieldSetting.name !== 'output'
) as ObjectFieldSetting[];
```

**Review by danvb112 — CHANGES_REQUESTED** [review]

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**